scripts/update_ml_tutorials.py

```diff
@@ -25,10 +25,10 @@
 
```

Comment:
Did anything relevant to the diff actually change in this PR? If not, please revert the changes to this file.

Reply:
uhm, never mind, I guess the our linter picks up this script? I see the same changes on https://github.com/HumanSignal/label-studio/pull/5686/commits/971b8b58e20a8da7c7135b554fd133e73055db90, and in any case nothing appears to be broken